activesupport/lib/active_support/core_ext/string/output_safety.rb

```diff
@@ -37,7 +37,7 @@ def unwrapped_html_escape(s) # :nodoc:
       if s.html_safe?
         s
       else
-        s.gsub(HTML_ESCAPE_REGEXP, HTML_ESCAPE)
+        ActiveSupport::Multibyte::Unicode.tidy_bytes(s).gsub(HTML_ESCAPE_REGEXP, HTML_ESCAPE)
```

Comment:
Do we want to? If anything I was thinking this should probably be applied to `html_escape_once`, too. The errors we're seeing come through from `TagHelper#tag_options`, which calls `ERB::Util.unwrapped_html_escape` directly.